config/defaults.conf

```diff
@@ -223,11 +223,18 @@ tokenizer = "MosesTokenizer"  // The name of the tokenizer, passed to the Task c
                               // Use "" for edge probing tasks to use the original tokenization,
                               // or one of 'OpenAI.BPE', 'MosesTokenizer', 'bert-uncased-base',
                               // etc.
-word_embs = none  // The type of word embedding layer. Set to none when using ELMo, GPT, or BERT.
-                  // Options: none, scratch (i.e., trained from scratch), glove, fastText.
+
+input_module = "" // The word embedding layer, available values are the following:
+                  // gpt, glove, ELMo, scratch (i.e., trained from scratch), fastText. For BERT, 
+                  // specify the version of BERT. Available values for BERT are: bert-base-uncased, 
+                  // bert-large-cased. For more, see:
+                  //   https://github.com/huggingface/pytorch-pretrained-BERT
+                  //   #loading-google-ais-pre-trained-weigths-and-pytorch-dump
+                  // For ELMo, BERT, and GPT, there are additional config parameters. 
+
```

Comment:
Mind adding some instructions here about tokenization support? (i.e., which of these require specific tokenization, and how that tokenization should be specified.)

Even better would be to have the option of leaving tokenizer blank, and having it be set to some sane default for each option.